Library.Business/DTO/Country/UpdateCountryDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Library.Business.DTO.Country
+{
+    public class UpdateCountryDto
```

Comment:
Why are additional models needed? )
It's good to have a single model for creating and updating

Reply:
Its what tearchers at IT Academy told us. Separate DTO for each action, if fields which is necessary for action is different